src/services/member/plugins/data/schemas/selects.ts

```diff
@@ -0,0 +1,47 @@
+import { Action } from '../../../../action/entities/action';
+import { ChatMessage } from '../../../../chat/chatMessage';
+import { ChatMention } from '../../../../chat/plugins/mentions/chatMention';
+import { Item } from '../../../../item/entities/Item';
+import { AppAction } from '../../../../item/plugins/app/appAction/appAction';
+import { AppData } from '../../../../item/plugins/app/appData/appData';
+import { AppSetting } from '../../../../item/plugins/app/appSetting/appSettings';
+import { ItemCategory } from '../../../../item/plugins/itemCategory/entities/ItemCategory';
+import { ItemFavorite } from '../../../../item/plugins/itemFavorite/entities/ItemFavorite';
+import { ItemLike } from '../../../../item/plugins/itemLike/itemLike';
+import { ItemMembership } from '../../../../itemMembership/entities/ItemMembership';
+import { schemaToSelectMapper } from '../utils/selection.utils';
+import {
+  actionSchema,
+  appActionSchema,
+  appDataSchema,
+  appSettingSchema,
+  itemCategorySchema,
+  itemFavoriteSchema,
+  itemLikeSchema,
+  itemMembershipSchema,
+  itemSchema,
+  messageMentionSchema,
+  messageSchema,
+} from './schemas';
+
+export const selectActions = schemaToSelectMapper<Action>(actionSchema);
+
+export const selectAppActions = schemaToSelectMapper<AppAction>(appActionSchema);
+
+export const selectAppSettings = schemaToSelectMapper<AppSetting>(appSettingSchema);
+
+export const selectAppData = schemaToSelectMapper<AppData>(appDataSchema);
```

Comment:
Nice way to reuse the schemas! Maybe I would simplify by not having this file, but creating the constants directly in the file they are used (the repositories). What do you think?

Reply:
Yes why not ! Indeed, because I don't reuse them their could be directly in the repositories.